docs/guide-uz/start-hello.md

```diff
@@ -92,15 +95,16 @@ http://hostname/index.php?r=site%2Fsay&message=Salom+dunyo
 Sizga "Salom, dunyo" matni chiqishi kerak. Header (yuqori bo'lim) va  Footer (pastki bo'lim) hamma sahifalardek chiqadi.
 Agar siz `message` kiritmasangiz, siz «Salom» matnini ko'rishingiz mumkin. Bunaqa mantiq chiqishi sababi shundagi, `actionSay` usuli (metod)dagi `message` parametri oldindan "Salom" matni berilgan.
 
-> Ma'lumot uchun: Yangi sahifa hamma ishlatayotgan Header (yuqori bo'limi) va Footer (pastki bo'lim) ishlatadi, chunki 
-  [[yii\web\Controller::render()|render()]] usuli `say` ko'rinishini o'zi [maketga](structure-views.md) `views/layouts/main.php`  topib qo'yadi.
+> Ma'lumot uchun: Yangi sahifa hamma ishlatayotgan Header (yuqori bo'limi) 
```

Comment:
Normally I wouldn't nag but since this is mainly reformatting - why is it broken here and why there is no indent before `views/layouts/main.php` while there are 2 spaces after it? And there is no indent in the next paragraph as well.

Reply:
I understand, I will check